packages/google_sign_in/google_sign_in_web/example/pubspec.yaml

```diff
@@ -2,8 +2,8 @@ name: google_sign_in_web_integration_tests
 publish_to: none
 
 environment:
-  sdk: ^3.3.0
-  flutter: ">=3.19.0"
+  sdk: ^3.4.0
+  flutter: ">=3.22.0"
```

Comment:
Why are a bunch of examples being given tighter SDK constraints than the library?

Reply:
No reason, I just didn't roll back the dependencies there, in case somebody uses the examples as copy-paste, at least they get "the latest".

Reply:
(Rolled back)